_posts/en/newsletters/2025-12-19-newsletter.md

```diff
@@ -0,0 +1,1246 @@
+- **Theoretical limitations on embedding data in the UTXO set:** ...
+
+{:#channeljamming}
+
+- **Channel jamming mitigation simulation results and updates:** Carla
+  Kirk-Cohen, in collaboration with Clara Shikhelman and elnosh, had posted the
+  [Lightning jamming simulation results][channel jamming results] for their
+  updated reputation algorithm. The updates included reputation tracking for
+  outgoing channels, and tracking incoming channel resource limitations. With
+  these new updates, they found that it still protects against
+  [resource][channel jamming resource] and [sink][channel jamming sink] attacks.
+  After this round of updates and simulations, they feel that [channel jamming
+  attack][channel jamming attack] mitigation has reached a point where it is
+  good enough.
+
+## November
+
+{:#secpperformance}
+
+- **Comparing performance of ECDSA signature validation in OpenSSL vs. libsecp256k1:** Sebastian Falbesoner conducted an
+  [analysis][openssl vs libsecp256k1] on the performance of ECDSA signature
+  validation between OpenSSL and libsecp256k1. Since 2015, Bitcoin Core has used
+  libsecp256k1 over OpenSSL. He wanted to be certain that doing so was the right
+  choice and not a wasted effort. Falbesoner found was that over the years,
+  libsecp256k1 had improved significantly, whereas OpenSSL had remained the
+  same. He also concluded that outside the Bitcoin ecosystem, the secp256k1
+  curve is not that relevant, so it is not justified for OpenSSL to put too many
+  resources into improving it (evident by the results).
+
+{:#restrictingdata}
+
+- **Multiple discussions about restricting data:** ...
+
+{:#stalerates}
+
+- **Modeling stale rates by propagation delay and mining centralization:**
+  Antoine Poinsot [posted][Antoine post] to Delving Bitcoin about modeling stale
+  block rates and how block propagation time affects a miner's revenue as a
+  function of its hashrate. In the post he setup a base-case scenario which
+  miners act realistically (default Bitcoin Core). This led to a revenue
+  proportional to their share of hashrate. He then outlines two situations in
+  which a block goes stale. The situations were either another miner found a
+  block before this miner or another miner found a block after this miner.
+  Poinsot pointed out that between these two situations a block is more likely
+  to become stale in the first one, he suggests that miners prefer to hear about
+  others' blocks faster than publishing their own. Later in the post he computes
+  by exactly how much does the probability increase and found that if a mining
+  operation with 5EH/s can expect a revenue of $91M and if blocks took 10
+  seconds to propagate the revenue would increase by $100k.
+
+{:#bip3}
+- **BIP3 and the BIP process:** In 2025, work on the Updated BIP Process
+  advanced significantly. BIP3 was [assigned][news341 bip3 assigned] a number in
+  January, published in February, and advanced to Proposed in April. Further
```

Comment:
Can link the published in February to our coverage https://bitcoinops.org/en/newsletters/2025/02/14/#updated-proposal-for-updated-bip-process